packages/icons/README.md

```diff
@@ -10,22 +10,22 @@ npm i pcln-icons
 
 ```js
 import React from 'react'
-import FlightsIcon from 'pcln-icons/lib/Flights'
+import { Flights as FlightsIcon } from 'pcln-icons'
 
 export default props => <FlightsIcon mr={2} />
 ```
 
 For a complete list of all icons, see: [the iconography docs](https://priceline.github.io/design-system/iconography)
 
-For backwards compatibility, the `Icon` component can be used in the same way as in the Design System v1.
-
-**Note:** The `Icon` component will be deprecated. Prefer using individual imports as shown above.
+**BREAKING CHANGE:** The `Icon` component has been removed from `core` (`pcln-design-system`) in v3.0.0. The optimal
+approach is to use named imports from `pcln-icons`. If you need an `<Icon>` component that supports a `name`
```

Comment:
Related to my previous comment - are we still technically allowing consumers to use `<Icon />` after v3 is published? Why do we want to support both? It feels a bit confusing.

Reply:
@abautist I agree, but it needs to exist for cases like amenities where the names of the icons come from GraphQL. `<Icon />` will only exist in `pcln-icons`, it has been removed from `core`

Reply:
It breaks tree-shaking Icons for the app, so it is less than ideal, but until teams have a chance to work around that (perhaps importing a limited set of icons), I didn't want to make the migration effort to v3 any more complicated than it needs to be.